tests/fullstack-test/expr/issue_7695.test

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mysql> drop table if exists test.t;
+mysql> create table test.t(a char(20));
+mysql> alter table test.t set tiflash replica 1
+func> wait_table test t
+
+mysql> insert into test.t values ('𞤀');
+mysql> set tidb_enforce_mpp=1; select a, lower(a) from test.t;
++------+----------+
+| a    | lower(a) |
++------+----------+
+| 𞤀     | 𞤢         |
++------+----------+
```

Comment:
It should be a display issue, the real result looks like this
<img width="462" alt="Screenshot 2023-10-25 at 17 53 19" src="https://github.com/pingcap/tiflash/assets/1916264/8bc82ec6-49d7-4639-a2c4-8c095fee195a">

Reply:
How about adding unit test instead of Integration  test?